### PR TITLE
Executing_Tests.rst: Improve getting started part

### PR DESCRIPTION
**Pull request by Makman2**

### CODE REVIEW

**Comment by Makman2** [comment]
This is more or less a copy of @jschwarzwalder 's commit from coala (with slight improvements on grammar and stuff).

File: Developers/Executing_Tests.rst [hunk 1]
```diff
@@ -9,26 +9,27 @@ Executing our Tests
...
+directory.
 
-If you have not already, clone the
-`repository <https://github.com/coala-analyzer/coala>`_ (or a fork of
-it) by running:
+If you have not already, you can start by cloning a copy of the project to
+your local machine.
+
+Either fork the repo on Github https://github.com/coala-analyzer/coala,
+or clone it directly from coala.
```
Comment (Makman2) [inline Developers/Executing_Tests.rst]: I think we should replace the complete git-clone stuff with a reference to another guide, like:

> To run tests, you need the source code of coala. You can get it by manually downloading from GitHub
> or cloning [<-- link to cloning tutorial] it from there.

File: Developers/Executing_Tests.rst [hunk 2]
```diff
@@ -9,26 +9,27 @@ Executing our Tests
 coala has a big test suite. It is meant to work on every platform on
 every PC. If you just execute our tests you are doing us a favor.
 
-To run tests, You first need to install some dependencies.
-This can be done by following these steps:
+To run tests, you will need to open a file located in the default coala
+directory.
```
Comment (Makman2) [inline Developers/Executing_Tests.rst]: too vague, which file, coala is not cloned yet (in principle). Also I don't recall someone needs to open some file :O

File: Developers/Executing_Tests.rst [hunk 3]
```diff
@@ -9,26 +9,27 @@ Executing our Tests
...
-Next you need to install some requirements. This can be
-done by executing the following command while in the root of the
-coala project directory.
+Next you need to install some dependencies. This can be
+done by executing:
 
 ::
 
-    $ pip3 install -r test-requirements.txt  -r requirements.txt
+    $ pip3 install -r test-requirements.txt -r requirements.txt
```
Comment (Makman2) [inline Developers/Executing_Tests.rst]: oh this is something that could go into an own commit...

Comment (Makman2) [inline Developers/Executing_Tests.rst]: issue opened for that^^

**Comment by Makman2** [comment]
I think the new commit does currently not add that much new information, the only step we can (and should maybe) include is a reference on how to get coala source code. But otherwise the instructions are already quite clear imo^^

**Comment by Adrianzatreanu** [comment]
coo, we should add the reference then :)

**Comment by sils1297** [comment]
@Makman2 this commit has valid review comments from you, why is it pending review?

**Comment by Makman2** [comment]
as this is not my commit, and I'm more or less about to reject it, I wanted to hear from someone else if this is okay^^
As we don't have a guide for "how to get coala source code", I would make an issue for that and close this PR.

**Comment by Makman2** [comment]
Issue opened for that here:
https://github.com/coala-analyzer/documentation/issues/106
